Jakefile.js

```diff
@@ -100,11 +100,20 @@ var serverSources = [
     "editorServices.ts",
     "protocol.d.ts",
     "session.ts",
+    "nodeimpl.ts",
     "server.ts"
 ].map(function (f) {
     return path.join(serverDirectory, f);
 });
 
+var languageServiceLibrarySources = [

```

Comment:
i would concat the services sources as well, to get incremental compilation working correctly.